resources/usuario.py

```diff
@@ -0,0 +1,48 @@
+from flask_restful import Resource, reqparse
+from models.usuario import UserModel
+class init(Resource):
```

Comment:
Choose a better descriptive name for this class. Also, use `PascalCase` convention.